src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.ts

```diff
@@ -34,7 +34,7 @@ export class CreateClusterComponent implements OnDestroy {
   currentStepSub: Subscription;
   permissions: Permissions;
   projectConstants: typeof AppConstants = AppConstants;
-  stepTitles = ['Add Hosts', 'Create OSDs', 'Review'];
+  stepTitles = ['Add Hosts', 'Create OSDs', 'Create Services', 'Review'];
```

Comment:
The initial agreement was to to use `Remove` with `Add` and `Delete` with `Create`. Since we are using Create and Delete from the beginning..so I decided to avoid refactoring unnecessary code and go on with `Create`